datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxHorizontalScalingService.java

```diff
@@ -99,24 +107,28 @@ public String triggerScalingFlow(SdxCluster sdxCluster, StackScaleV4Request scal
     }
 
     public void validateHorizontalScaleRequest(SdxCluster sdxCluster, DatalakeHorizontalScaleRequest request) {
-        if (!ENTERPRISE.equals(sdxCluster.getClusterShape())) {
-            throw new BadRequestException(String.format("scaling not supported on: %s. " +
-                    "Only the ENTERPRISE Datalake shape support the horizontal scaling functionality.", sdxCluster.getClusterShape().name()));
+        if (ENTERPRISE != sdxCluster.getClusterShape()) {
+            throw new BadRequestException(String.format("scaling not supported on: %s.", sdxCluster.getClusterShape().name()));
```

Comment:
"Horizontal scaling not supported on {sdx.getClusterShape()}., please use ENTERPRISE Datalake shape."

Reply:
+1